PWGCF/TwoParticleCorrelations/Tasks/corrSparse.cxx

```diff
@@ -14,27 +14,65 @@
 /// \author Thor Jensen (thor.kjaersgaard.jensen@cern.ch) and Debojit Sarkar (debojit.sarkar@cern.ch)
 
 #include <vector>
+
 #include "Framework/runDataProcessing.h"
 #include "Framework/AnalysisTask.h"
 #include "Framework/AnalysisDataModel.h"
 #include "Framework/ASoAHelpers.h"
+#include "Framework/ASoA.h"
 #include "Framework/HistogramRegistry.h"
 #include "Framework/RunningWorkflowInfo.h"
 #include "CommonConstants/MathConstants.h"
+#include "CCDB/BasicCCDBManager.h"
+#include "Common/Core/RecoDecay.h"
+
 #include "Common/DataModel/EventSelection.h"
 #include "Common/DataModel/TrackSelectionTables.h"
 #include "Common/DataModel/Centrality.h"
 #include "Common/DataModel/Multiplicity.h"
+#include "PWGCF/DataModel/CorrelationsDerived.h"
 #include "PWGCF/Core/CorrelationContainer.h"
 #include "PWGCF/Core/PairCuts.h"
-#include "Common/Core/RecoDecay.h"
+
+namespace o2::aod
+{
+namespace multiplicity_nch
```

Comment:
I would suggest a name more adapted to your analysis to avoid clashes 
What about `corrsparse`?